database/dataloaderSegmentation.py

```diff
@@ -0,0 +1,52 @@
+
+'''
+Creating a Dataloader for segmentation taks
+'''
+
+import numpy as np
+import os
+from PIL import Image
+from torch.utils.data import Dataset
+
+
+class DatasetLoaderSegmentation(Dataset):
+    ''' Derived class to manage Segmentation Dataset Image Loading
+    '''
+
+    def __init__(self, root,
```

Comment:
I don't think that we need root, images, targets, targetext.

We can simplify using wildcard and 2 parameters:
 
* `image_path` without any default. It can take a path as `'/some/path/*.png'`
* `label_path` without any default. It can take a path as `'/some/path/*.png'`

Reply:
label is better than target for segmentation I think